release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -586,6 +586,15 @@ You can decrease power consumption through the Node Tuning Operator by specifyin
 [id="ocp-4-11-auth"]
 === Authentication and authorization
 
+[id="ocp-4-11-post-installation"]
+=== Post-installation configuration
+
+[id="ocp-4-11-cluster-capabilities"]
+==== Cluster capabilities
+As a cluster administrator, you can enable cluster capabilities to select or deselect one or more optional components prior to installation or post installation.
+
+For more information, see cluster Capabilities.
```

Comment:
Link will be updated after the https://github.com/openshift/openshift-docs/pull/47915 is merged.